src/System Application/Test Library/FormatValidation/src/FormatValidationTestTable.table.al

```diff
@@ -0,0 +1,33 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+// namespace ?
+
+table 130443 "Format Validation Test Table"
+{
+    Caption = 'Format Validation Test Table';
+    DataClassification = SystemMetadata;
+
+    fields
```

Comment:
It is not necessary. In the unit test codeunit it is also possible to assign a value without using a table. Please discuss it internally because this was the suggestion from your colleagues.

Reply:
I think the suggestion came from @pri-kise, who is one of our most active GitHub contributors and a valued MVP, but he is not a colleague :-) Be that as it may, I value everyone's input on equal terms. Let me think about this for a minute.

Reply:
I wouldn't use here a test table just for this and send sql queries.
If we would keep it, I would than think about using it as Temp table. 

Anyway, not really for it, from the tests written it's just un-necessary to fill the table then to test. Its a simple test, wouldn't overcomplicate this.

Reply:
Changed it to temp table.